src/map/utils/battleutils.cpp

```diff
@@ -5379,9 +5379,7 @@ namespace battleutils
 
     int32 BreathDmgTaken(CBattleEntity* PDefender, int32 damage)
     {
-        float resist = 1.0f + PDefender->getMod(Mod::UDMGBREATH) / 10000.f;
-        resist       = std::max(resist, 0.f);
-        damage       = (int32)(damage * resist);
+        float resist = 1.0f;
 
         resist = 1.0f + PDefender->getMod(Mod::DMGBREATH) / 10000.f + PDefender->getMod(Mod::DMG) / 10000.f;
```

Comment:
You could make all of these either be `float resist;` or `float resist = 1.0f + PDefender->getMod(Mod::DMGBREATH) / 10000.f + PDefender->getMod(Mod::DMG) / 10000.f;`. As it stands assigning it to `1.0f` doesn't actually do anything.

Reply:
Adjusted Frank's PR for him to be the 2nd suggestion.